src/patch.jl

```diff
@@ -45,6 +45,46 @@ end
 
 PatchEnv(debug::Bool=false) = PatchEnv(Dict{Any,Vector{Function}}(), debug)
 
+function patches(pe::PatchEnv)
+    #! format: off
+    return [
+        Patch(target, func)
+        for (target, alternate_funcs) in pe.mapping
+        for func in alternate_funcs
+    ]
+    #! format: on
+end
+
```

Comment:
Should be able to remove this with the other suggestion:
```suggestion
```

Reply:
(done)